src/ActionsTests/Actions/ScreenShotActionUnitTests.cs

```diff
@@ -17,6 +18,17 @@ namespace Axe.Windows.ActionsTests.Actions
     [TestClass]
     public class ScreenShotActionUnitTests
     {
+        // This sets the Bounding Rectangle. It breaks encapsulation in
+        // many reproachable ways
```

Comment:
>many reproachable ways [](start = 11, length = 22)

@RobGallo, any thoughts on cleaner options here? BoundingRect has no setter

Reply:
This would be slightly cleaner if it were an extension method. But I would only implement the extension method in the unit test project, which probably means it isn't worth it.

Reply:
That was more or less my logic. It's only used in this class so I scoped it as such

---
In reply to: [289187766](https://github.com/microsoft/axe-windows/pull/37#discussion_r289187766) [](ancestors = 289187766)